.prettierignore

```diff
@@ -24,3 +24,6 @@ yarn-error.log*
 dist
 .solid
 pnpm-lock.yaml
+
+.svelte-kit
+.vscode
```

Comment:
any particular reason for this?

Reply:
The .svelte-kit folder is auto generated by SvelteKit and should not be checked by prettier. I guess the .vscode one is unnecessary